src/main/java/org/jabref/gui/cleanup/FieldFormatterCleanupsPanel.java

```diff
@@ -115,7 +115,7 @@ private void buildLayout() {
 
         BibDatabaseContext databaseContext = JabRefGUI.getMainFrame().getCurrentBasePanel().getBibDatabaseContext();
 
-        recommendButton = new Button(Localization.lang("Recommended for %0", databaseContext.getMode().getFormattedName()));
+        recommendButton = new Button(Localization.lang("Reset to recommended", databaseContext.getMode().getFormattedName()));
         boolean isBiblatex = databaseContext.isBiblatexMode();
```

Comment:
Please move this line in the `setOnAction` block so that changes to the mode are reflected in the buttons behavior.

Reply:
Sorry, I don't quite get it. Do you mean because of the change of name? This is why I renamed the button so there is no reference to the used mode. Or do you mean something else?

Reply:
Sorry for being imprecise here. In fact, I don't know what I meant either 😸